### DISCUSSION TITLE
Dokładna popularność aukcji

### DISCUSSION THREAD

**Discussion by blonders** [allegro/allegro-api#7192] [Suggestions - rejected]
Witam 

Proszę o rozważenie dodania do API informacji o popularności aukcji i nie chodzi o zakres od - do tak jak jest w przypadku 'offers/listing' tylko o realne wartości jak na froncie. Po uruchomieniu zagranicznego serwisu każdy serwis ma własne wartości dla popularności co jest logiczne i fajnie byłoby mieć wgląd do tych danych dla każdego marketplace osobno i tu moja sugestia aby np w sale/offers dodać :
"stats" -> "popularity": 124; ( to było by dla marketplace "allegro-pl" ) 
"additionalMarketplaces" -> "allegro-cz" -> "stats": ->"popularity": 9; (i to było by dla marketplace "allegro-cz" )

i/lub do sale/offers/{offerId} dodać na podobnych zasadach :) 

Pozdrawiam.

**Comment by gsti** [comment]
Kiedyś były te informacje jeszcze przed marketplace "allegro-cz" - popieram ten pomysł w 100%. 

P.S. Żadnej odpowiedzi od tygodnia? :(

**Comment by PrzemyslawLukanowski** [comment]
W zakładce Discussions zbieramy sugestie, które dotyczą rozwoju API. Nie odpowiadamy na każdą z nich, zainteresowanie daną zmianą widzimy na podstawie liczby oddanych głosów przez innych użytkowników.

Jeśli chodzi o zasób /sale/offers, jest to odpowiednik zakładki "Mój Asortyment". Zwracamy w nim wyłącznie liczbę sprzedanych sztuk w ciągu ostatnich 30 dni - zarówno dla serwisu bazowego (w polu `stock.sold`), jak i serwisu dodatkowego (w polu `additionalMarketplaces.{addtionalMarketplace}.stock.sold`). Na tę chwilę nie planujemy wprowadzać dodatkowych zmian w tym zakresie. Jeśli jednak miałoby się coś zmienić w tym temacie - poinformujemy o tym w naszym newsie oraz w tym wątku.
